modules/packages/Curl.chpl

```diff
@@ -36,711 +38,1040 @@ The `curl homepage <https://curl.haxx.se/libcurl/>`_ describes libcurl thus::
 Dependencies
 ------------
 
-The Curl module in Chapel is dependent on libcurl. For more information
- on how to setup libcurl, see the :ref:`readme-auxIO` page.
+The Curl module in Chapel is dependent on libcurl being installed and available
+to Chapel pograms.
```

Comment:
Can you run `chplspell` on this PR?

Reply:
Does the user need to specify the include and library flags or do we do some of that in require statements already? 

I always appreciate an example compilation command for reference.